README.md

```diff
@@ -153,15 +153,15 @@ Certain functions in ODK Collect depend on cloud services that require API keys
 
 **Google Maps API**: When the "Google Maps SDK" option is selected in the "User interface" settings, ODK Collect uses the Google Maps API for displaying maps in the geospatial widgets (GeoPoint, GeoTrace, and GeoShape).  To enable this API:
   1. [Get a Google Maps API key](https://developers.google.com/maps/documentation/android-api/signup).  Note that this requires a credit card number, though the card will not be charged immediately; some free API usage is permitted.  You should carefully read the terms before providing a credit card number.
-  1. Edit or create `collect_app/secrets.properties` and set the `GOOGLE_MAPS_API_KEY` property to your API key.  You should end up with a line that looks like this:
```

Comment:
There's no need `secrets.properties` to be in `collect_app`. Seems more intuitive to me for it to live in the root.

Reply:
It will be annoying to have to remember to move it and I guarantee some people who maintain forks (maybe even me!) will be super confused and have a hard time finding this note. Does it really seem worth it? I don’t feel strongly but do want to make sure you’ve considered that this has some costs for some types of users.

Reply:
Yeah that's a good point. The main reason for moving it was that `secrets.gradle` needs to access it, and it felt more natural for it to live in the top instead of having to reach down into a project. I'll think of a way to flag it during build (so it's obvious that `secrets.properties` needs to move) or move it back.